jenkins-slaves/ansible/kie-rhel7.yaml

```diff
@@ -0,0 +1,364 @@
+---
+- hosts: packer
+  remote_user: root
+  gather_facts: false
+  vars :
+    maven_versions: [3.0.5, 3.2.3, 3.2.5, 3.3.9, 3.5.0, 3.5.2]
+    jdk_download_urls: [
+      "https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/slaves-resources/lastSuccessfulBuild/artifact/jdk-6u45-linux-x64.tar.gz",
+      "https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/slaves-resources/lastSuccessfulBuild/artifact/jdk-7u80-linux-x64.tar.gz",
+      "https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/slaves-resources/lastSuccessfulBuild/artifact/jdk-8u152-linux-x64.tar.gz",
+      "https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/slaves-resources/lastSuccessfulBuild/artifact/jdk-9.0.1-linux-x64.tar.gz"
+    ]
+    jdk_short_versions: [6u45, 7u80, 8u152, 9.0.1]
+    jdk_long_versions: [jdk1.6.0_45, jdk1.7.0_80, jdk1.8.0_152, jdk-9.0.1]
+    jdk_symlink_names: [jdk1.6, jdk1.7, jdk1.8, jdk9]
+    firefox_short_versions: [38esr, 45esr, 52esr]
+    firefox_long_versions: [38.8.0esr, 45.5.0esr, 52.0.2esr]
+
+  tasks:
+  - name: Copy Pulp repos config
+    copy:
+      src: data/config/pulp/pulp.repo
+      dest: /etc/yum.repos.d/pulp.repo
+      owner: root
+      group: root
+      mode: 0644
+
+  - name: Install EPEL7 repository
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: Import EPEL7 repository key
+    shell: rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
+
+  - name: Update all packages
+    yum:
+      name: "*"
+      state: latest
+
+  - name: Add user jenkins
+    user:
+      name: jenkins
+      # droolsjbpm.ci1
+      password: $6$rounds=656000$skw/fVgypJZWP8m1$04UK6Ax4.2hhJfcCbgGegNGl0brQtufYJd4lGn4Lw2RnZ8GQL50crLAd/PIU/.JC047TLybYfcdLL/27hkwlx0
```

Comment:
The problem I see here is that salt will prevent only for rainbow table attacks. If this is a dictionary based password it still could be "recovered" without enormous computer power.

Reply:
The only reason the password is not in a clear-text is that ansible does not like that (the password is actually in a clear text just line above that). If we are concerned about security, then I would disable password login completely and just rely on the ssh keys.

Reply:
I don't know if we should be concerned about security. I probably pay too much attention to that. The problem is that addressing security concerns would decrease convenience of usage but on the other hand if we don't see security risk it doesn't mean that it doesn't exist. It's always trade off.

I would leave it as it is if people use this access method. If it's rarely used I would probably remove it.